core.ghactivity.php

```diff
@@ -1034,5 +1037,144 @@ public static function count_repos( $date_start, $date_end, $person = '' ) {
 
 		return (int) count( $repos );
 	}
+
+	/**
+	 * Record any label updates into taxonomy meta of issue post.
+	 *
+	 * @since 2.1
+	 */
+	public function update_issue_labels() {
+		$event_list = $this->get_github_issue_events();
+
+		if ( ! isset( $event_list ) || ! is_array( $event_list ) ) {
+			return;
+		}
+
+		// Sorts all the events by created date from older to newer.
+		usort($event_list, function( $a, $b ) {
```

Comment:
Since PHP 5.2 is still supported by WordPress, I would recommend that you move this to a separate function, like I did with `sort_totals`.